### ISSUE TITLE
Selecting LZMA compression causes the ISO to not boot.

### ISSUE THREAD

**Issue by Augusto7743** [PJ-Singh-001/Cubic#145]
Hello.
I want recompress the OS original ISO to an better compression.
Using Cubic current version I had done test with Lubuntu 18.04.5 32 bits compress the ISO using LZMA and XZ.
ISO extracted and only selected the compression level and after create the ISO.

From 1.1 GB to
- XZ > 979
- LZMA > 981

However when testing in an VirtualBox VM the created ISO using LZMA not does boot.
Is displayed the OS boot menu, but when selecting "Start Installer" and process stop with message

    initramfs //filesystem.squashfs failed invalid argument
    can not mount /dev/loop0

I have done another test with another ISO Lubuntu 20.04.5 64 bits and happen the same issue above.

I had accessed the link below about XZ
https://www.nongnu.org/lzip/xz_inadequate.html

Really the information make sense ... I have avoided XZ for backup for it.

Only reporting the bug and not understand if is an bug when creating the ISO using LZMA or Ubuntu not allow boot ISO with squash using LZMA.

**Comment by PJ-Singh-001** [comment]
Augusto,

Thanks for reporting.
I will test this on my end.

**Comment by Augusto7743** [comment]
I need share another information.
I have used Cubic to create some custom ISOs with xz compression ... sometimes when in ISO boot in moment when is doing "file check" happen an checksum "error".
The strange detail is if doing an machine reset the file check is completed without errors.
USB port and driver without any problem.
That strange false checksum error had happened sometimes ... after I had accessed the link
https://www.nongnu.org/lzip/xz_inadequate.html
Have an section saying about checksum errors.
I not understand if is related with the false checksum error. However the information about xz issues make sense.

Perhaps ISO created using LZMA not load the squashfs because not is enabled to boot using LZMA ?

**Comment by PJ-Singh-001** [comment]
I do get the same error.

![image](https://user-images.githubusercontent.com/19394936/229388819-104a861d-f4a0-4587-bca3-e6937baa5348.png)

While `lzma` and `xz` use the same compression algorithm, `xz` is considered better/newer and `lzma` is considered legacy.

For now, I'll leave the `lzma` option on the [Compression page](https://github.com/PJ-Singh-001/Cubic/wiki/Compression-Page). I may consider removing it in a future release.

**Comment by PJ-Singh-001** [comment]
I am going to remove the `lzma` option from the [Compression page](https://github.com/PJ-Singh-001/Cubic/wiki/Compression-Page).

**Comment by PJ-Singh-001** [comment]
Here is the new page...

![Cubic Compression Page](https://user-images.githubusercontent.com/19394936/229935751-e1b4396e-2f5f-46b5-b614-53a6d0d72ed0.png)

**Comment by PJ-Singh-001** [comment]
Fix released in Cubic version 2023.04.80.

**Comment by Augusto7743** [comment]
@PJ-Singh-001

I had created an topic in Ubuntu forums about that issue.
An forum user had replied

"I looked it up... People have been not able to boot squashfs compressed with LZMA since before 2008... People have wondered since then, why LZMA is even an option. Not sure."

https://ubuntuforums.org/showthread.php?t=2483048&goto=newpost

Yes ... XZ use LZMA algorithm.
However is good read the informations in the link below
https://www.nongnu.org/lzip/xz_inadequate.html
